assets/license-headers-check

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env bash
+#
+# Iterates over all rust source files and ensures they start with the license
```

Comment:
Shouldn't we make htis work for all source files we work with? `*.rs`, `*.js`, `*.svelte`, `*.ts`, ~~*.hs~~ #criesinpain, etc?

Reply:
Yes please, contributions welcome :smile: